purchase_order_secondary_unit/tests/test_purchase_order_secondary_unit.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields
+from odoo.tests import SavepointCase
+
+
+class TestPurchaseOrderSecondaryUnit(SavepointCase):
+    at_install = False
```

Comment:
PLZ remove `at_install`and `post_install` https://github.com/OCA/odoo-community.org/issues/28